stock_landed_cost_company_percentage/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_landed_cost_company_line,access_landed_cost_company_line,model_landed_cost_company_line,stock.group_stock_manager,1,1,1,1
```

Comment:
Does it work when logged with a non manager user? I'd think they'd need a read access on the model

Reply:
Corresponded. Thanks.